C/guide/ch_txns.xml

```diff
@@ -336,7 +336,7 @@
             account name and &app; will fill in the remaining part of the name. Typing the separator
             character at any time will complete the current level of the account name, leaving the
             cursor positioned to start the next level of account name. For example, typing
-            <keycap>A:C</keycap> might complete to the <emphasis>Assets:Checking</emphasis> account.
+            <guilabel>A:C</guilabel> might complete to the <emphasis>Assets:Checking</emphasis> account.
```

Comment:
Fixed in commit 24a91f0. Perhaps you want to rebase this PR?

Reply:
> (But, by the way, maybe you have known, it's unreasonable to use `<keycombo>` to instruct Chinese reader how to enter characters, because there is no correspondence between a single Chinese character and a single physical keycap. So I will express this meaning with `<emphasis>X:Y</emphasis>` in translated texts.)

I know almost nothing about input methods for CJK scripts,  Perhaps you could create a wiki page —and link it in https://wiki.gnucash.org/wiki/Documentation_Improvement and other I18N/L10N pages—

1. explainig the differences compared with latin scripts
2. develope a rule set for the portation.  BTW: How is it solved in ja/? Tip: call `kdiff3 zh/guide/index.dokbook ja/guide/index.dokbook` to compare them.

Reply:
@fellen, Japanese and Chinese input methods work similarly. The main difference is that Japanese has an alphabetic form called [Hiragana](https://en.wikipedia.org/wiki/Hiragana) (it actually has two but the other one, [Katakana](https://en.wikipedia.org/wiki/Katakana), is used for foreign words that don't generally map to Chinese characters, called Kanji in Japanese) so a Japanese using a Japanese keyboard will type in Hiragana or using a Roman keyboard in a literation called [Romaji](https://en.wikipedia.org/wiki/Romaji) while a Chinese will type in a Romanization, nearly always [Pinyin](https://en.wikipedia.org/wiki/Pinyin). The IME will use the entry to present a list of possible character completions in a drop-down list that changes as more letters are typed, somewhat like, though vastly more complex than, the typeahead search in the register.

Korean is written in an alpha-phonic system called [Hangul](https://en.wikipedia.org/wiki/Hangul) so the input method is a simple keyboard mapping. 

> call kdiff3 zh/guide/index.dokbook ja/guide/index.dokbook to compare them.

Since the Japanese translation is now almost 12 years old that's very unlikely to produce anything useful.

Reply:
Japanese is based on the Chinese character, but using phonetic symbols for publishment is acceptable. So, you can input Chinese character used in Japanese with phonetic symbols.

Chinese character used in Chinese can be input via phonetic symbols (Pinyin in Mainland, Zhuyin in Taiwan) or structure components (Wubi/Zhengma in Mainland, Cangjie in Taiwan), which both are mapped to US keyboard.

Modern Korean has dropped the Chinese character. The Korean character only shows exact pronunciation and is just like what you write down phonemes into a small square area. You can imagine a keyboard layout to input international phonemes.

Modern Vietnamese is another language that has dropped the Chinese character. It has its own alaphbet which also represents a pronunciation rule like Korean, but its writting is just like Latin - one letter by one letter from left to right.

Reply:
I compiled [GnuCash Wiki:CJK Input Methods](https://wiki.gnucash.org/wiki/CJK_Input_Methods) from your comments. Feel free to improve the page!